### ISSUE TITLE
Tag blacklist

### ISSUE THREAD

**Issue by Aidolii** [johanneszab/TumblThree#241]
There's currently a way to download only from certain tags, but no way to prevent certain tags from downloading. There's a few artists I follow whom I like most of the content of, but every so often they'll post something I find a little disturbing. Said posts are tagged properly, so it would be easy to prevent if TumblThree had a blacklist.

**Comment by johanneszab** [comment]
No, it's not possible right now. But it's a good idea and easy to implement. Shouldn't be more than a line a code.

**Comment by Aidolii** [comment]
Maybe even a line of code that can be included along with the (hopefully) upcoming video download fix? 😉

**Comment by Aidolii** [comment]
I've come across an art blog that posts other people's art with the tag 'reblog' (without using the reblog system). The problem is they don't have a tag they use specifically for their own art.

A blacklist to get rid of the 'reblog' tag would be very useful here.
